adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -473,6 +507,25 @@ def connect_AP(self, ssid, password): # pylint: disable=invalid-name
             raise RuntimeError("No such ssid", ssid)
         raise RuntimeError("Unknown error 0x%02X" % stat)
 
+    def create_ap(self, ssid, password, channel=b'\x01'):
```

Comment:
I don't really like `channel` as a `byte`. Consider changing it to an `int` and then converting it into a byte within `create_ap`, along w/checking if it's btween 1 and 14.